lib/util/time.ex

```diff
@@ -147,4 +133,28 @@ defmodule Util.Time do
   def time_of_day_add_minutes(time_of_day, minutes) do
     time_of_day + 60 * minutes
   end
+
+  @doc """
+  Converts a timestamp into a Date.
+
+  iex> Util.Time.date_of_timestamp(1546362000)
+  ~D[2019-01-01]
+  """
+  @spec date_of_timestamp(timestamp()) :: Date.t()
+  def date_of_timestamp(timestamp) do
```

Comment:
#107 will have a merge conflict with this, since it also makes the same function. (No action needed here. This comment is for my own reference.)